<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AvidMech Trade Login</string>
  </property>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>560</y>
     <width>330</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Koodak</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enter">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>510</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Koodak</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>enter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_User">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>305</y>
     <width>330</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Koodak</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>please enter your username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Pass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>385</y>
     <width>330</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Koodak</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="placeholderText">
    <string>please enter your password</string>
   </property>
  </widget>
  <widget class="QLabel" name="User_Icon_Label">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>317</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Forget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>330</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Koodak</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>forget password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Logo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>230</width>
     <height>230</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Pass_Icon_Label">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>398</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
